docs/.vuepress/config.js

```diff
@@ -79,16 +79,18 @@ module.exports = {
               '/introduction/faq'
             ]
           },
+          {
+            title: 'Install Filecoin',
+            path: '/install-filecoin',
+            children: [
+              '/install-filecoin/requirements',
+              '/install-filecoin/install'
+            ]
+          },
```

Comment:
I think this install section should have the perma url `/install/`, then we have the flexibility to add different methods in the future `/install/lotus/` etc.

Reply:
Not sure what you mean here. Can you make a suggestion?

Reply:
Nevermind, just seen [your other comment](https://github.com/filecoin-project/filecoin-docs/pull/235#pullrequestreview-466828209).

Reply:
Fixed here:

![image](https://user-images.githubusercontent.com/9611008/90177298-ba566d00-dd80-11ea-81c6-e262886b04de.png)